elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala

```diff
@@ -175,18 +176,18 @@ class RestlasticSearchClient(endpointProvider: EndpointProvider, signer: Option[
     runEsCommand(query, s"/${index.name}/${tpe.name}/_query", DELETE)
   }
 
-  def startScrollRequest(index: Index, tpe: Type, query: QueryRoot, resultWindow: String = "1m"): Future[ScrollId] = {
+  def startScrollRequest(index: Index, tpe: Type, query: QueryRoot, resultWindowOpt: Option[String] = None, fromOpt: Option[Int] = None, sizeOpt: Option[Int] = None): Future[(ScrollId, SearchResponse)] = {
```

Comment:
I personally consider it awkward to return a tuple.  It produces more easily read consumer code if it returns a case class.  Specifically, `._1` and `._2` have little meaning, but `.scrollID` and `.searchResponse` would have more context.

Some internal teams have adopted never using tuples as return types in a public API